README.md

```diff
@@ -1,2 +1,12 @@
 # medicine_project
 Program to help you monitor your medication use
+
+## Setup
+
+1. Make sure you have Python set up, preferably in a [virtualenv](https://virtualenv.pypa.io/en/latest/user_guide.html)
+2. Run `pip install -e .`
+3. Run `medicine` or `python src/medicine/main.py`
+
+## Tests
+
+Tests can be run with `pytest`
```

Comment:
można ustawić by testy się opdalały automatycznie przy każdej zmianie kodu na githubie, co jest spoko

Reply:
terminal to twój przyjaciel. Trochę może byc przerażający na początku, ale korzystaj z niego grubo i gęsto. Można skonfigurować editora by to tez zrobił, ale na tym się nie znam - osobiście w konsoli zawsze se odpalam, podobnie gita se ręcznie robię w konsoli. Ale to kwestia gustu - jak jest dobrze skonfigurowany, to poprostu będzie działał. W tym oczywiście cały problem...

Testy z założenia są w osobnym pliku, i są osobnymi bytami. Testujesz se kod jakkolwiek ci jest wygodniej (osobiśice polecam mieć odpalony ipython i se wklekać tam kawałki kodu), a testy piszesz jak już masz względnie gotowe funkcje. Takie testy są owszem uciążliwe, ale to się bardzo opłaca